web/mixins/FcMixinRequestStudyLeaveGuard.js

```diff
@@ -26,7 +26,7 @@ export default {
     },
     actionNavigateBack(leaveConfirmed = false) {
       this.leaveConfirmed = leaveConfirmed;
-      this.$router.push(this.$refs.nav.routeNavigateBack);
+      this.$router.push(this.routeNavigateBack);
```

Comment:
if we got rid of the `FcDialogConfirmRequestStudyLeave` import and just let the components that use this mixin provide their own dialogs, this would be a generic mixin that we could use elsewhere (e.g. in View Reports)